tests/augmentations/test_coarsedropout.py

```diff
@@ -0,0 +1,35 @@
+import pytest
+import numpy as np
+
+from discolight.annotations import (annotations_to_numpy_array)
+from discolight.augmentations.coarsedropout import CoarseDropout
+
+
+@pytest.mark.usefixtures("sample_image")
+def test_coarsedropout(sample_image):
+
+    img, annotations = sample_image
+
+    bboxes = annotations_to_numpy_array(annotations)
+
+    augmentation = CoarseDropout(0.1, 25)
```

Comment:
Since you are constructing the augmentation directly, the parameters must be named:

```suggestion
    augmentation = CoarseDropout(p=0.1, num=25)
```